OCP-Infrastructure/kafka/01_kafka.yml

```diff
@@ -52,29 +51,27 @@ spec:
         port: 9094
         tls: true
         type: route
-    replicas: 3
+    replicas: 1
     storage:
-#      class: gp2
-      size: 50Gi
+      size: 0.5Gi
```

Comment:
läuft gerade ein kafka cluster? falls ja, wird diese Änderung nicht problemlos funktionieren, weil PVC downsizing nicht möglich ist.

Reply:
Die habe ich schon eingespielt. Mir ist nichts aufgefallen, aber ich habe auch zwischendurch die PVCs gelöscht. Bei Hogarama ist das nicht kritisch, da die Daten nur temporär in Kafka benötigt werden.